shared/actions/json/waiting.json

```diff
@@ -8,6 +7,9 @@
       "key": ["string", "Array<string>"],
       "error?": "RPCError"
     },
+    "batchChangeWaiting": {
+      "changes": "Array<{key: string | Array<string>, increment: boolean, error?: RPCError}>"
+    },
```

Comment:
looks like changeWaiting is only used in one place (engine below) so we can get rid of it

Reply:
good idea, removing